docker/genericsql/Dockerfile

```diff
@@ -45,6 +45,19 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 && rm freetds-1.1.39.tar.gz \
 && rm -rf freetds-1.1.39
 
+# Install the Microsoft ODBC driver for SQL Server (Linux)
+RUN apt-get update \
+&& apt-get install -y --allow-downgrades gpgv=2.2.12-1+deb10u1 gnupg2 curl apt-utils \
+&& curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - \
+&& curl https://packages.microsoft.com/config/debian/10/prod.list > /etc/apt/sources.list.d/mssql-release.list \
+&& apt-get update \
+&& ACCEPT_EULA=Y apt-get install msodbcsql17 \
+&& ACCEPT_EULA=Y apt-get install mssql-tools \
+&& echo 'export PATH="$PATH:/opt/mssql-tools/bin"' >> ~/.bash_profile \
```

Comment:
Not sure what this is for. But it probably doesn't have the effect you want as we don't load the bash profile when running integrations.